Anytype/Sources/ApplicationLayer/AppSceneHandler/AppSceneHandlerModifier.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import SwiftUI
+
+private struct AppSceneHandlerModifier: ViewModifier {
+    
+    @StateObject private var model = AppSceneHandlerModifierModel()
+    
+    func body(content: Content) -> some View {
+        content
+            .onOpenURL { url in
+                model.onOpenURL(url)
+            }
+    }
```

Comment:
🔥🔥🔥

Reply:
We can

Reply:
We can't because safari should be open from current screen. This is global handler for external deep links and universal links

Reply:
But i can try to join this logic and URLViewModifier in next pr

Reply:
i found some bug and fixed